reconcile/queries.py

```diff
@@ -9,13 +9,16 @@
 
 from jinja2 import Template
 
+from reconcile.gql_definitions.common.smtp_client_settings import (
+    query as smtp_client_config_query,
+    SmtpSettingsV1,
+)
```

Comment:
I am wondering if we should start to cut `queries.py` into smaller files, mainly because I am concerned about duplicate names of generated objects.

An example of what I mean can be seen here: https://github.com/fishi0x01/qontract-reconcile/blob/demo-solution/reconcile/queries.py#L11-L20

```python
from reconcile.gql_definitions.cluster_peering.queries.cluster_peering import (
    ClusterV1,
    DEFINITION as CLUSTER_PEERING_QUERY,
    query as cluster_peerings_query
)
from reconcile.gql_definitions.clusters.clusters import (
  ClusterV1 as OtherClusterV1,
  DEFINITION as CLUSTERS_MINIMAL_QUERY,
  query as minimal_cluster_query,
)
```

Multiple functions return lists of `ClusterV1`, but from different queries. Maybe we should make a module `typed_queries` and have smaller files in it, each dedicated to a single query, in order to have smaller scopes and avoid naming collisions? Curious of your thoughts here.

Reply:
I like this idea!